tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/js/operator-finder/operator-finder.js

```diff
@@ -126,7 +127,80 @@ define(['jquery', 'lodash', 'log', 'handlebar', 'designViewUtils', 'app/source-e
+         * @returns {*{}} return extension object which contains converted md
+         */
+        var mdConversion = function (cloneOperator) {
+            //Add remarkable instance to convert md type data into html
+            var markDownConvertor = new Remarkable({
+                html: true, // Enable HTML tags in source
+                xhtmlOut: false, // Use '/' to close single tags (<br />)
+                breaks: false, // Convert '\n' in paragraphs into <br>
+                linkify: true, // Autoconvert URL-like text to links
+                // Enable some language-neutral replacement + quotes beautification
+                typographer: false,
+                // Double + single quotes replacement pairs, when typographer enabled,
+                // and smartquotes on. Set doubles to '«»' for Russian, '„“' for German.
+                quotes: '“”‘’',
+                highlight: function (/*str, lang*/) {
+                    return '';
+                }
+            });
+            if (cloneOperator.description) {
+                cloneOperator.extensionDescription = markDownConvertor.render(cloneOperator.description);
+            }
+            if (cloneOperator.examples) {
+                cloneOperator.example = "";
+                cloneOperator.examples.forEach(function (e, i) {
+                    //change the "|" as "," and "\n" as "<br/> in returnAttributes description to avoid md
+                    // conversation bug.
+                    e.syntax = pipelineNewlineEscape(e.syntax);
+                    cloneOperator.example += "<h5>example " + (++i) + "</h5>"
+                        + "<pre>" + e.syntax + "</pre>" + "<p>" + e.description + "</p>";
+                });
+                cloneOperator.example = markDownConvertor.render(cloneOperator.example);
+            }
+            if (cloneOperator.parameters) {
+                cloneOperator.parameterTable = "| Name | Possible DataTypes | Description | " +
+                    "Default Value | Optional | Dynamic | " + "\n" +
+                    "| ------| ------| -----------|" +
+                    " ------| ------| ------|\n";
+                cloneOperator.parameters.forEach(function (m) {
+                    //change the "|" as "," and "\n" as "<br/> in returnAttributes description to avoid md
+                    // conversation bug.
+                    m.description = pipelineNewlineEscape(m.description);
+                    cloneOperator.parameterTable += " | " + m.name + " | " + m.type.join('<br/>') + " | " +
+                        m.description + " | " + m.defaultValue + " | " + m.optional + " | "
+                        + m.dynamic + " | " + "\n";
+                });
+                cloneOperator.parameterTable = markDownConvertor.render(cloneOperator.parameterTable);
+            }
+            if (cloneOperator.returnAttributes) {
+                cloneOperator.returnTable = "| Name | DataTypes | Description |\n " +
```

Comment:
Name is only available in some of the functions, do a check and do not add name column if this is not used. Refer to the old code where it compiled retyurn type tables

Reply:
960249df47be3bbdc76d0016d7e1dd9ac324ac05 this commit fix above request change